src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -37,17 +39,27 @@ export default function FacilityAddress({
           <br />
         </>
       )}
-      {!!phone && (
-        <dl className="vads-u-margin-y--0">
-          <dt className="vads-u-display--inline">
-            <strong>Main phone:</strong>
-          </dt>{' '}
-          <dd className="vads-u-display--inline">
+      {isV2 &&
+        !!phone && (
+          <div className="vads-u-margin-top--1p5">
+            <strong>Clinic: </strong> {clinicFriendlyName}
             <br />
+            <strong>Main phone: </strong>
             <FacilityPhone contact={phone} />
-          </dd>
-        </dl>
-      )}
+          </div>
+        )}
+      {!isV2 &&
+        !!phone && (
+          <dl className="vads-u-margin-y--0">
+            <dt className="vads-u-display--inline">
+              <strong>Main phone:</strong>
+            </dt>{' '}
+            <dd className="vads-u-display--inline">
+              <br />
```

Comment:
If this is deliberate for spacing, can we use a class instead?

Reply:
@jbalboni - What's the best way to address this?

Reply:
It's sort of in a weird spot, but it's not for spacing, it's to create a line break between the label and the content here.

I think I'd rather leave this the way it is for now, if we want to change it I'd want to look into our other uses of `dl` elements and why we've marked this one as an inline element, too. I can make a ticket for that.

Reply:
Thanks.